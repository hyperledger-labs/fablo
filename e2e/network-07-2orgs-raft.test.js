const { performTests } = require('./performTests');

const label = 'network-07-2orgs-raft';

describe(label, () => {
  performTests(label, 'fabrikkaConfig-2orgs-2channels-2chaincodes-tls-raft.json');
});
