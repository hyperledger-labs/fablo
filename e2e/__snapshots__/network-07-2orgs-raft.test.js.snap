// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`samples/fablo-config-hlf2-2orgs-2chaincodes-raft.yaml should create proper files from samples/fablo-config-hlf2-2orgs-2chaincodes-raft.yaml 1`] = `Array []`;
